src/components/toast/toast-theme.scss

```diff
@@ -1,15 +1,20 @@
 md-toast.md-THEME_NAME-theme {
+
   .md-toast-content {
     background-color: #323232;
     color: '{{background-50}}';
 
     .md-button {
       color: '{{background-50}}';
+
       &.md-highlight {
-        color: '{{primary-A200}}';
-        &.md-accent {
-          color: '{{accent-A200}}';
+        // By default the toast should use the accent color as the primary highlight color
+        color: '{{accent-A200}}';
```

Comment:
@DevVersion This is a breaking change since we are changing the default from primary to accent. I believe it is a correct breaking change, but we need to make sure that we note it in the commit message (and PR description) so that users will know. Can you update the commit message accordingly?